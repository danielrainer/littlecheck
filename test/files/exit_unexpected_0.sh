# RUN: /bin/sh %s
# CHECKEXIT: 1
