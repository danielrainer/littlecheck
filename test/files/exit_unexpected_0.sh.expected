Failure in exit_unexpected_0.sh:

  Process unexpectedly exited with code 0 (expected: 1)

  when running command:
    /bin/sh exit_unexpected_0.sh
